scripts/release/release.sh

```diff
@@ -57,8 +71,8 @@ function changelogMain {
 }
 
 function modifyVersionFiles {
-  sed -i "s/var SDKVersion =.*/var SDKVersion = \"${TARGET_VERSION}\"/" meta/meta.go
-  sed -i "s/var SDKPrerelease =.*/var SDKPrerelease = \"\"/" meta/meta.go
+  sed -i "s/var SDKVersion =.*/var SDKVersion = \"${TARGET_VERSION_CORE}\"/" meta/meta.go
```

Comment:
I think this PR looks good, can you cleanup the changes done for testing and squash the commits?